client/src/plugins/update-checks/UpdateChecks.js

```diff
@@ -73,6 +73,8 @@ export default class UpdateChecks extends PureComponent {
 
       const updateCheckInfo = await config.get(UPDATE_CHECKS_CONFIG_KEY);
 
+      updateCheckInfo.stagedRollout = false;
```

Comment:
```suggestion
      const updateCheckInfo = await config.get(UPDATE_CHECKS_CONFIG_KEY) || {}

      updateCheckInfo.stagedRollout = false;
```

Cf. https://github.com/camunda/camunda-modeler/pull/2395/files#r690955107

Reply:
ty

Reply:
Do we monkey patch the persistent configuration here? For me it looks like it and I'd rather prefer to inject stagedRollout in a safe manner, i.e. 

```
updateCheckInfo = { ..., stagedRollout: false };
```